manifest/openshift4/coreos/ncp-openshift4.yaml

```diff
@@ -438,6 +452,58 @@ data:
 
 
 
+---
+apiVersion: v1
+kind: SecurityContextConstraints
+metadata:
+  annotations:
+    kubernetes.io/description: scc-ncp provides the SCC for NCP and NSX Node Agent
+  creationTimestamp: null
+  name: ncp-scc
+
+allowHostDirVolumePlugin: true
+allowHostIPC: false
+allowHostNetwork: true
+allowHostPID: true
+allowHostPorts: false
+allowPrivilegedContainer: true
+allowedCapabilities:
+- SYS_ADMIN
+- NET_ADMIN
+- NET_RAW
+- SYS_PTRACE
+- DAC_READ_SEARCH
+- SYS_NICE
+- SYS_MODULE
+
+defaultAddCapabilities: null
+fsGroup:
+  type: MustRunAs
+groups:
+- system:authenticated
+
+priority: null
+readOnlyRootFilesystem: false
+requiredDropCapabilities:
+- KILL
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  seLinuxOptions:
+    type: spc_t
+    level: s0:c0.c1023
+  type: MustRunAs
+supplementalGroups:
+  type: RunAsAny
+volumes:
+- configMap
+- downwardAPI
+- emptyDir
+- persistentVolumeClaim
+- projected
+- secret
+
```

Comment:
sorry I forgot to tell you that this should be removed. It's stale because it's for ocp3.

Reply:
Thanks for reminder. corrected.